tutorials/spectral/plot_SpectralVariance.py

```diff
@@ -18,7 +18,8 @@
 ###################################################################################################
 
 # Import spectral module
-from neurodsp import spectral
+from neurodsp.spectral import (compute_spectral_hist, compute_spectrum, compute_scv,
```

Comment:
Super nitpick: but can you 'balance' lines by putting 2 functions on line 1 and 2 on line 2 (also groups scv funcs together).

Reply:
Fixed.